sessions/recap-project-6-NEW/activity-planner/user-stories/1_activities-list.md

```diff
@@ -0,0 +1,47 @@
+# Activities List
+
+## Value Proposition
+
+**As a user**,
+
+**I want to** browse a list of activities,
+
+**so that** I can easily find and explore new activities that interest me.
+
+## Description
+
+![wireframe](./assets/activities-list.png)
+
+## Acceptance Criteria
+
+- The homepage displays a list of activities fetched from MongoDB.
+- The activities list is headlined with its purpose.
+- Each activity listing includes:
+  - Activity title
+  - Activity image
+  - Activity categories
+- The list supports vertical scrolling to accommodate multiple entries.
+- Activities are fetched from the MongoDB database.
+
+## Tasks
+
+- [ ] Create feature branch `feature/activities-list`
+- [ ] Set up MongoDB connection in the project
+  - [ ] Set up a new cluster
+  - [ ] Configure environment variables for MongoDB connection
+- [ ] Create Activity model with Mongoose schema:
+  ```javascript
+  {
+    title: { type: String, required: true },
+    imageUrl: { type: String, required: true },
+    categories: [{ type: String, required: true }],
+    description: String,
+    area: String,
+    country: String,
+  }
+  ```
+- [ ] Create API endpoint to fetch activities
+- [ ] Implement server-side sorting (newest first)
+- [ ] Add error handling for database connection issues
+- [ ] Add loading states for data fetching
+- [ ] Add error states for failed data fetching
```

Comment:
these seem to be duplicates from the first user story. Is it possible that you copied the old user stories without adapting the AC / Tasks?